readme.md

```diff
@@ -7,15 +7,21 @@ The user documentation for QHAna can be viewed on [qhana.readthedocs.io](https:/
 
 ## Start
 
-`docker-compose up`
+`docker compose up`
 
-If you want to use the [NISQ-Analyzer](https://github.com/UST-QuAntiL/nisq-analyzer) plugin, use `docker-compose --profile nisq up`.
+If you want to use the [NISQ-Analyzer](https://github.com/UST-QuAntiL/nisq-analyzer) plugin, use `docker compose --profile nisq up`.
 
 ## Using QHAna
 
 Open http://localhost:8080 in a web browser to use the QHAna UI.
 
 
+## Proxy configuration
+
+To temporarily add more ports to the proxy configuration, set the environment variable `EXTRA_PROXY_PORTS` to the additional ports e.g. `:1234 :2345`.
```

Comment:
Add a short explanation for what the proxy is for. For example something like:
The QHAna docker containers all contain a proxy that maps localhost ports back to the actual localhost running docker.
This simplifies networking when part of the stack is executed locally.

Also mention the list of ports that are already mapped to localhost. And maybe mention that 8080 cannot be used as the port is used in the containers already.